src/main/java/org/jetbrains/research/anticopypaster/models/UserSettingsModel.java

```diff
@@ -0,0 +1,18 @@
+package org.jetbrains.research.anticopypaster.models;
+
+import org.jetbrains.research.extractMethod.metrics.features.FeaturesVector;
+
+public class UserSettingsModel extends PredictionModel{
+
```

Comment:
I would add a way to instantiate it and have the instantiation be creating the metricgatherer and doing nothing else, for now. Later on it'll pass that to the other classes but for now just instantiating it here would be good.

Reply:
I pushed off the instantiation to the model, but we need to figure out testing before uncommenting the instantiation lines since the gatherMetrics() call will cause the tests to fail.